srcpkgs/urxvt-font-size/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'urxvt-font-size'
+pkgname=urxvt-font-size
+version=1.3
+revision=1
+archs="i686 x86_64"
```

Comment:
Why is this set?

Reply:
Wasn't entirely sure if required... I now see it's unnecessary from the packaging manual.

Amended the original commit.